client/src/views/Main.tsx

```diff
@@ -14,8 +14,8 @@ const {
   CookiePolicy,
   ForgotPassword,
   SetNewPassword,
-  Login,
-  Signup,
+  // Login,
+  // Signup,
```

Comment:
We can probably remove these comments right?

Reply:
removed